tests/unit/components/Collections/ListItems/PlioListItem.spec.js

```diff
@@ -714,7 +783,7 @@ describe("PlioListItem.vue", () => {
     // click the delete button
     await wrapper
       .get('[data-test="optionDropdown"]')
-      .findAll('[data-test="option"]')[4]
+      .findAll('[data-test="option"]')[5]
```

Comment:
Isnt there a better way to do this? Maybe finding by the exact type of option?
Because everytime something will be added into the dropdown list, all of these values will need to change